README.md

```diff
@@ -213,6 +237,6 @@ We love to share our community spotlights across various platforms! You can find
 
 - LinkedIn: [marimo LinkedIn](https://www.linkedin.com/company/marimo-io/)
 - Twitter: [@marimo_io](https://x.com/marimo_io)
-- Discord: [marimo community](https://discord.gg/JE7nhX6mD8)
+- Discord: [marimo community](https://discord.gg/QdpFxJWhyt?ref=spotlights)
```

Comment:
you can change this to `https://marimo.io/discord?ref=spotlights`. in case the discord link changes again

Reply:
Yeah, did that. Thanks for spotting that.